base/run.sh

```diff
@@ -20,6 +20,16 @@ if [[ $GITREPO != "" ]]; then
   cd /home/$NEWUSER && /usr/bin/git clone $GITREPO
 fi
 
+# Configure git username and email so that we don't have to do it every time
+echo "[user]" >> /home/$NEWUSER/.gitconfig
+if [[ $GITUSER != "" ]]; then
```

Comment:
/nitpick (don't bother unless you feel like it)

Rather test `$GITUSER == ""`, and then assign the value of $NEWUSER to it. There is some fancy bash syntax for doing this, but I'm not a fan because of readability.

Then you'll just need to echo into `.gitconfig` once.